configure.ac

```diff
@@ -307,6 +307,19 @@ AC_ARG_WITH([profiler],
 )
 AC_MSG_RESULT([$with_profiler])
 
+# System luajit
+AC_MSG_CHECKING([whether to enable system luajit])
+AC_ARG_ENABLE([system-luajit],
```

Comment:
This looks odd, shouldn't that be AC_ARG_WITH ? Alternatively, the help text ought to say --enable-system-luajit, right? I think in general, libraries are specified with --with- options, right? E.g.

```
--with-system-luajit=/opt/luajit
```

Wonder if "system" is redundant here in that case? But I realize this patch doesn't support this, for now, but it'd be nice not having to change the configure option if someone adds support for this.